typescript/packages/subsurface-viewer/package.json

```diff
@@ -1,37 +1,50 @@
 {
-    "name": "@webviz/subsurface-viewer",
-    "version": "0.0.0",
-    "description": "",
-    "main": "dist/index.js",
-    "scripts": {
-        "transpile": "tsc --project ./tsconfig.json",
-        "copy-files": "copyfiles package.json dist && copyfiles README.md dist && copyfiles --exclude **/*.{tsx,jsx,js,ts,snap} --up 1 \"src/**/*\" dist/",
-        "build": "npm run transpile && npm run copy-files"
-    },
-    "author": "Equinor <opensource@equinor.com>",
-    "license": "MPL-2.0",
-    "dependencies": {
-        "@deck.gl/core": "^8.8.25",
-        "@emerson-eps/color-tables": "^0.4.57",
-        "@equinor/eds-core-react": "^0.16.1",
-        "@equinor/eds-icons": "^0.19.1",
-        "@mui/material": "^5.13.4",
-        "@mui/system": "^5.12.1",
-        "@nebula.gl/layers": "^1.0.4",
-        "@reduxjs/toolkit": "^1.7.2",
-        "@vivaxy/png": "^1.3.0",
-        "ajv": "^7.2.1",
-        "convert-units": "^2.3.4",
-        "d3": "^7.8.2",
-        "d3-color": "^3.1.0",
-        "d3-format": "^1.4.5",
-        "deck.gl": "^8.9.19",
-        "gl-matrix": "^3.4.3",
-        "lodash": "^4.17.21",
-        "mathjs": "^9.4.2",
-        "react-redux": "^7.2.6"
-    },
-    "files": [
-        "/dist"
-    ]
-}
+  "name": "@webviz/subsurface-viewer",
+  "version": "0.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "transpile": "tsc --project ./tsconfig.json",
+    "build": "git clean -xdff dist && npm run transpile"
```

Comment:
Not sure, but I think you need the `copy-files` to copy some asset files.

Reply:
It has been added to subsurface-viewer and wll-log-viewer, which are the 2 only projects that have png files.